newton/_src/solvers/implicit_mpm/rasterized_collisions.py

```
@@ -75,97 +85,120 @@ class Collider:
 
     min_sdf = ground_sdf
     sdf_grad = collider.ground_normal
     sdf_vel = wp.vec3(0.0)
+    closest_point = wp.vec3(0.0)
     collider_id = int(_GROUND_COLLIDER_ID)
+    material_id = int(_GROUND_COLLIDER_MATERIAL_INDEX)
 
     # Find closest collider
-    for m in range(collider.meshes.shape[0]):
-        mesh = collider.meshes[m]
+    global_face_id = int(0)
```

Ruff (0.14.2)

Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)